Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCrowdStrikeFalcon.yaml

```diff
@@ -0,0 +1,296 @@
+        "2176", "Prevention-quarantine,blocked ", "Deny", "Failure",
+        "2304", "Detection", "Allow", "Success",
+        "2432", "Detection-quarantine", "Allow", "Success",
+        "4096", "Prevention-blocked", "Deny", "Failure",
+        "4112", "Prevention-blocked,killed", "Deny", "Failure",
+        "4224", "Prevention-blocked,quarantine", "Deny", "Failure",
+        "4240", "Prevention-killed,blocked,quarantine", "Deny", "Failure",
+        "4352", "Detection", "Allow", "Success",
+        "4368", "Detection", "Allow", "Success",
+        "4638", "Detection", "Allow", "Success",
+        "5120", "Prevention-blocked", "Deny", "Failure",
+        "8192", "Disabled", "Allow", "Success",
+        "8208", "Detection", "Allow", "Success",
+        "8320", "Detection-quarnatine", "Allow", "Success",
+        "8704", "Detection", "Allow", "Success",
+        "9216", "Detection", "Allow", "Success",
+        "10240", "Detection", "Allow", "Success",
+        "12304", "Detection", "Allow", "Success",
+        "16400", "Killed", "Deny", "Failure",
+        "32768", "Prevention-blocked", "Deny", "Failure",
+        "32896", "Prevention-blocked,quarantine", "Deny", "Failure",
+        "33024", "Detection", "Allow", "Success",
+        "65536", "Downgraded", "Allow", "Success",
+        "65552", "Prevention-killed", "Deny", "Failure",
+        "65792", "Detection-downgraded", "Allow", "Success",
+        "65808", "Detection-downgraded", "Allow", "Success",
+        "73728", "Detection-downgraded", "Allow", "Success",
+        "73744", "Detection-downgraded", "Allow", "Success",
+        "131088", "Prevention-killed", "Deny", "Failure",
+        "131216", "Prevention-killed,quarantine", "Deny", "Failure",
+        "131584", "Prevention-killed", "Deny", "Failure",
+        "131712", "Prevention-killed,quarantine", "Deny", "Failure",
+        "2099200", "Prevention-blocked", "Deny", "Failure",
+        "2099328", "Prevention-blocked,quarantine", "Deny", "Failure",
+        "4196352", "Prevention-blocked", "Deny", "Failure",
+        "4196480", "Prevention-blocked,quarantine", "Deny", "Failure",
+        "1048576", "Prevention-suspend", "Deny", "Failure",
+        "524288", "Prevention-suspend", "Deny", "Failure",
+        "262144", "Blocking Disabled", "Allow", "Success",
+        "16384", "Safeguard Enabled", "Allow", "Success",
+        "131072", "Kill Failed", "Deny", "Failure",
+        "256", "Policy Disabled", "Allow", "Success",
+        "2097152", "Response Action Already Applied", "Deny", "Failure",
+        "4194304", "Response Failed", "Deny", "Failure"
+    ];
+    let parser = (disabled: bool=false) {
+        let alldata = CommonSecurityLog
+            | where not(disabled)
+                and DeviceVendor == "CrowdStrike"
+                and DeviceProduct == "FalconHost"
+            | where DeviceEventClassID in ("Network Access In A Detection Summary Event", "FirewallMatchEvent");
```

Comment:
@niralishah-crest @jayeshprajapaticrest 
Hope we validated Network related logs? fall under these two categories only